Gopkg.toml

```diff
@@ -81,3 +81,7 @@
   go-tests = true
   non-go = true
   unused-packages = true
+
+[[constraint]]
```

Comment:
I suggest you move this section before `[prune]`.

Reply:
updated